project/src/js/ui/tournament/tournamentXhr.js

```diff
@@ -9,5 +9,17 @@ export function tournament(id) {
 }
 
 export function reload(id) {
-  return request(`/tournament/${id}`, { background: true });
+  return request('/tournament/' + id, { background: true });
+}
+
+export function resync(id) {
```

Comment:
you can't send hardcoded socketVersion. And `resync` is not useful anymore I believe. Check lila code or ask @ornicar